stacks/victoria-metrics-cluster/deploy-local.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
Can you make the `deploy-local.sh` a part of the `deploy.sh` file. 
eg - [deploy.sh](https://github.com/digitalocean/marketplace-kubernetes/blob/master/stacks/kube-prometheus-stack/deploy.sh)

Reply:
Hi @ keladhruv ! 
Done